src/frequenz/sdk/timeseries/ev_charger_pool/_state_tracker.py

```diff
@@ -0,0 +1,172 @@
+# License: MIT
+# Copyright © 2022 Frequenz Energy-as-a-Service GmbH
+
+"""State tracking for ev charger pools."""
```

Comment:
General suggestion, please capitalize `ev` if it is used in the documentation and it isn't part of a variable/attribute name

Reply:
please check update all occurrences from ` ev ` to ` EV ` if this suggestion is taken

Reply:
thanks,  I've fixed all occurrences.

Reply:
Thanks!